Accelerometer.js

```diff
@@ -1,78 +0,0 @@
-import React, { useState, useEffect } from 'react';
```

Comment:
Deleted file containing imports and logic relevant to accessing mobile devices' accelerometer.

This file has been supplanted by `useIsShake.js` for further development and implementing device shake and animation triggering.